lib/run-status.js

```diff
@@ -166,6 +172,7 @@ RunStatus.prototype.processResults = function (results) {
 	this.tests = flatten(this.tests);
 
 	this.passCount = sum(this.stats, 'passCount');
+	this.knownFailureCount = sum(this.stats, 'knownFailureCount');
```

Comment:
This doesn't seem to work for me when I run your code against a test file (will post the code in a PR comment).